src/Test/intlinear.jl

```diff
@@ -681,6 +681,305 @@ function indicator4_test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+function semiconttest(model::MOI.ModelLike, config::TestConfig{T}) where T
+    atol = config.atol
+    rtol = config.rtol
+
+    @test MOIU.supports_default_copy_to(model, #=copy_names=# false)
+    @test MOI.supports(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{T}}())
+    @test MOI.supports(model, MOI.ObjectiveSense())
+    @test MOI.supports_constraint(model, MOI.SingleVariable, MOI.Semicontinuous{T})
+
+    # 2 variables
+    # min  x
+    # st   x >= y
+    #      x ∈ {0.0} U [2.0,3.0]
+    #      y = 0.0
+
+    MOI.empty!(model)
+    @test MOI.is_empty(model)
+
+    v = MOI.add_variables(model, 2)
+    @test MOI.get(model, MOI.NumberOfVariables()) == 2
+
+    vc1 = MOI.add_constraint(model, MOI.SingleVariable(v[1]), MOI.Semicontinuous(T(2), T(3)))
+    @test MOI.get(model, MOI.NumberOfConstraints{MOI.SingleVariable,MOI.Semicontinuous{T}}()) == 1
+
+    vc2 = MOI.add_constraint(model, MOI.SingleVariable(v[2]), MOI.EqualTo(zero(T)))
+    @test MOI.get(model, MOI.NumberOfConstraints{MOI.SingleVariable,MOI.EqualTo{T}}()) == 1
+
+    cf = MOI.ScalarAffineFunction{T}(MOI.ScalarAffineTerm{T}.([one(T), -one(T)], v), zero(T))
+    c = MOI.add_constraint(model, cf, MOI.GreaterThan(zero(T)))
+    @test MOI.get(model, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T},MOI.GreaterThan{T}}()) == 1
+
+    objf = MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1.0, 0.0], v), 0.0)
+    MOI.set(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), objf)
+    MOI.set(model, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+
+    @test MOI.get(model, MOI.ObjectiveSense()) == MOI.MIN_SENSE
+
+    if config.solve
+        @test MOI.get(model, MOI.TerminationStatus()) == MOI.OPTIMIZE_NOT_CALLED
+
+        MOI.optimize!(model)
+
+        @test MOI.get(model, MOI.TerminationStatus()) == config.optimal_status
+
```

Comment:
Can you remove the empty lines in between these lines to be consistent with other tests ?

Reply:
You would like to remove only the lines between the `MOI.optimize!(model) `and `@test MOI.get(model, MOI.TerminationStatus()) == config.optimal_status` (lines 725,747, ...) or all empty lines in the `if config.solve` block (723, 725,727, ...) ?
I don't like these empty lines too, but I was trying to make it look the same as the other tests, like in lines 363, 303, ...

Reply:
You can leave 723 and 725 bu remove 727, 729, ..., 737